unit_test/fuzzing/test_transport/test_spdm_transport_mctp_encode_message/transport_mctp_encode_message.c

```diff
@@ -39,7 +43,10 @@ void libspdm_test_transport_mctp_encode_message(void **State)
                              32; /* MCTP_MAX_RANDOM_NUMBER_COUNT */
     LIBSPDM_ASSERT(spdm_test_context->test_buffer_size > record_header_max_size);
 
-    transport_message_size = spdm_test_context->test_buffer_size - record_header_max_size;
+    alignment = MCTP_ALIGNMENT;
+    aligned_message_size = (spdm_test_context->test_buffer_size + (alignment - 1)) &
+                           ~(alignment - 1);
+    transport_message_size = aligned_message_size;
```

Comment:
Please explain why.

Reply:
There are alignment operations in the function
https://github.com/DMTF/libspdm/blob/345e2f866ca463f989343a126e7beb6a563d7c31/library/spdm_transport_pcidoe_lib/libspdm_doe_pcidoe.c#L71-L76